ara/views.py

```diff
@@ -74,17 +74,30 @@ def playbook(playbook):
 
 
 @app.route('/playbook/<playbook>/host/<host>')
-def playbook_host(playbook, host):
+@app.route('/playbook/<playbook>/host/<host>/status/<status>')
```

Comment:
But don't try to fix that now.  I've submitted #33 as a suggested modification of this PR; it's mostly just a reduction in code through the use of macros.  I haven't attempted any filtering changes.